app/components/FormLine.js

```diff
@@ -9,15 +9,14 @@ const FormLine = ({ label, value, valueUpdateCB }) => {
   //console.log('VALUE UPDATE CB IN FORM LINE ' + typeof valueUpdateCB);
   const removeNonNumericValues = (label, value) => {
     if (!value) {
-      return;
-    }
-
-    const regexNotNumberOrDot = /[^0-9.]/g;
-    const regexLeadingNumber = /^0+/;
-
-    value.replace(regexNotNumberOrDot, "");
-    value.replace(regexLeadingNumber, "");
+      value = 0;
```

Comment:
without setting 'value = 0', you cannot backspace to an empty input field.
The .replace functions didn't work since you weren't resetting the value.